package.json

```diff
@@ -340,7 +340,7 @@
   },
   "license": "AGPL-3.0",
   "name": "twenty",
-  "packageManager": "yarn@4.3.1",
+  "packageManager": "yarn@4.4.0",
```

Comment:
nope nope :p

Reply:
I think we should upgrade at some point but we should use the occasion to upgrade node at the same time :)
Let's keep it as it was